addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -125,13 +125,13 @@ private _id = switch (_type) do {
     };
     case "group": {
         if (_applyRetroactively) then {
-            [GVAR(oldUnit), group GVAR(oldUnit), grpNull] call _function;
+            [GVAR(oldUnit), grpNull, group GVAR(oldUnit)] call _function; // intentionally reversed order for backwards compatiblity
         };
         [QGVAR(groupEvent), _function] call CBA_fnc_addEventHandler // return id
     };
     case "leader": {
         if (_applyRetroactively) then {
-            [GVAR(oldUnit), leader GVAR(oldUnit), objNull] call _function;
+            [GVAR(oldUnit), objNull, leader GVAR(oldUnit)] call _function; // intentionally reversed order for backwards compatiblity
```

Comment:
Well, this is not the old/bwc behavior. I would report the new state on both args to be safe.

Reply:
old way was mixed
retro retruned current
```
[GVAR(oldUnit), group GVAR(oldUnit)] call _function;
```
but playerEH returned old because the assignment came after the event
```
[QGVAR(groupEvent), [_player, GVAR(oldGroup)]] call CBA_fnc_localEvent;
GVAR(oldGroup) = _data;
```